forms/common-themes/src/main/resources/theme/base/admin/resources/js/services.js

```diff
@@ -900,14 +900,15 @@ module.factory('PasswordPolicy', function() {
     var p = {};
 
     p.policyMessages = {
-        hashIterations: "Number of hashing iterations.  Default is 1.  Recommended is 50000.",
-        length:         "Minimal password length (integer type). Default value is 8.",
-        digits:         "Minimal number (integer type) of digits in password. Default value is 1.",
-        lowerCase:      "Minimal number (integer type) of lowercase characters in password. Default value is 1.",
-        upperCase:      "Minimal number (integer type) of uppercase characters in password. Default value is 1.",
-        specialChars:   "Minimal number (integer type) of special characters in password. Default value is 1.",
-        notUsername:    "Block passwords that are equal to the username",
-        regexPatterns:  "Block passwords that do not match all of the regex patterns (string type)."
+        hashIterations: 	"Number of hashing iterations.  Default is 1.  Recommended is 50000.",
+        length:         	"Minimal password length (integer type). Default value is 8.",
+        digits:         	"Minimal number (integer type) of digits in password. Default value is 1.",
+        lowerCase:      	"Minimal number (integer type) of lowercase characters in password. Default value is 1.",
+        upperCase:      	"Minimal number (integer type) of uppercase characters in password. Default value is 1.",
+        specialChars:   	"Minimal number (integer type) of special characters in password. Default value is 1.",
+        notUsername:    	"Block passwords that are equal to the username",
+        regexPatterns:  	"Block passwords that do not match all of the regex patterns (string type).",
+        expiredPasswords:  	"Minimal number (integer type) to block resetting to n old passwords. Default value is 3."
```

Comment:
expiredPasswords is not clear. I'd change it to "passwordHistory" and the text to "Block passwords that are equal to previous passwords. Default value is 3."